cluster/src/main/scala/org/apache/spark/executor/SnappyExecutor.scala

```diff
@@ -108,6 +108,24 @@ class SnappyExecutor(
       }
     }
   }
+
+  override def isStoreCloseException(t : Throwable) : Boolean = {
+    try {
+      Misc.checkIfCacheClosing(t)
+    } catch {
+      case ex: CacheClosedException => true
+      case _ => false
+    }
+    return false;
```

Comment:
Instead of "return false" the false can be moved below Misc.checkIfCacheClosing:
```scala
try {
  Misc.checkIfCacheClosing(t)
  false
} ...
```